README.md

```diff
@@ -14,3 +14,13 @@ and searchable in ways other than direct file reading:
 It's not stable yet, but the intention is to get a stable database
 schema, as well as a stable and specified schema of documents served
 by the web API.
+
+## Setup
+
+You will need to install a few perl packages to get the scripts running.
+
+For Arch Linux you will need the following packages: `perl-dbd-sqlite` (which also pulls the needed `perl-dbi`), `perl-cgi`, `perl-xml-libxml` and `perl-xml-libxslt`.
+
+You need to clone the lyrics repo into the folder above lyrics-api.
```

Comment:
That's a rather quick hack there, perhaps we should fix it rather than to document. Although documenting current state of things and changing it once we'll add nice argument parsing would work too.

Reply:
No, I don't think clonning the repo automatically would be nice. But I used a hardcoded path there, which isn't nice either: would be better to get that as an argument.

Reply:
Same applies to DB paths in both scripts, by the way: things like that should be configurable.